tsconfig.json

```diff
@@ -1,18 +1,58 @@
 {
-  "compilerOptions": {
-    "types": ["@types/node"],
-    "noEmit": true,
-    "allowJs": true
-  },
   "extends": "./tsconfig.base.json",
-  "include": [
-    "typings",
-    "tools/**/*.ts",
-    "tools/**/*.mts",
-    "eslint.config.js",
-    "eslint.config.mjs",
-    "jest.config.base.js",
-    "jest.config.js",
-    "jest.preset.js"
+  "compileOnSave": false,
+  "files": [],
+  "references": [
+    {
+      "path": "./packages/ast-spec"
+    },
+    {
+      "path": "./packages/eslint-plugin"
+    },
```

Comment:
One gotcha here may be that TS won't do rebuilds in some situations if _all_ projects are not listed here. Right now, you're using this top-level "solution" that points to other "solutions". Related is: https://github.com/microsoft/TypeScript/issues/30608 / https://github.com/microsoft/TypeScript/pull/55339